src/main/java/com/techiteasy/demo/models/Television.java

```diff
@@ -0,0 +1,202 @@
+package com.techiteasy.demo.models;
+//De 'models' vertegenwoordigen de data-objecten die worden opgeslagen in de database. Deze klasse kan gebruikt worden om een tabel te genereren in de database via JPA.
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.time.LocalDate;
+
+//@Entity markeert deze klasse als een JPA-entiteit (wat betekent dat deze overeenkomst met een tabel in de database)
+@Entity
+public class Television {
+
+    @Id //Dit attribuut geeft aan dat het veld 'id' de primaire sleutel is.
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    //Deze annotatie zorgt ervoor dat de waarde van 'id' automatisch wordt gegenereerd door de database wanneer een nieuwe rij wordt ingevoegd
+    private long id;
```

Comment:
Het is beter om hier het type "Long" te gebruiken in plaats van "long". Het zal nooit een probleem opleveren, omdat deze waarde verplicht is, maar primitieve waarden kunnen wel problemen opleveren in combinatie met JPA. De voorkeur gaat dus altijd uit naar de wrapper klassen (de hoofdletter variant)